src/components/Core/ImdbButton/style.js

```diff
@@ -0,0 +1,15 @@
+import { Button } from "@mui/material";
+import { styled } from "@mui/system";
+
+export const StyledButton = styled(Button)(({ theme, isYellow, padding }) => ({
+  cursor: "pointer",
+  paddingTop: "4px",
+  paddingBottom: "4px",
+  paddingLeft: padding,
+  paddingRight: padding,
+  color: isYellow ? "#000" : "#5799ef",
+  backgroundColor: isYellow ? "#f5c518" : "#1F1F1F",
```

Comment:
also let the user to pass the backgroundColor through props, to make sure that the color is determined just do this : 
backgroundColor : backgroundColor || "#1F1F1F"